### PR TITLE
Fix FileMan Bugs #284

### LINKED ISSUE #284: FileMan Bugs
### Describe the bug
1. Does not work for Windows. The file separator does not match. Everywhere, we have mentioned "/", which is Unix Styled. We need to instead put `os.PathSeparator`
2. There is a small but dangerous bug. If we specify the filenames as `"/../"`, it allows us to insert into the parent folder, which is dangerous. We must use absolute paths to check.

>If this bug restricts your use of space-cloud, give it a thumbs up reaction, so that we can determine which bugs need to be fixed immediately.
>👍

### PR DESCRIPTION
**Pull request by Inconnu08**
- Fixes #284
- Addes corresponding test
- Fixes typos
- Bumps the version
- Ran Gofmt

### CODE REVIEW

File: modules/auth/file_test.go [hunk 1]
```diff
@@ -25,10 +26,10 @@ func TestGetFileRule(t *testing.T) {
...
-		{testName: "Success", path: "/folder", module: &Module{fileRules: map[string]*config.FileRule{"create": fileRule1, "delete": fileRule1, "read": fileRule1}}},
-		{testName: "Success", path: "/folder/file", module: &Module{fileRules: map[string]*config.FileRule{"create": fileRule1, "delete": fileRule1, "read": fileRule1}}},
+		{testName: "Success", path: ps, module: &Module{fileRules: map[string]*config.FileRule{"create": fileRule, "delete": fileRule, "read": fileRule}}},
+		{testName: "Success", path: ps + "folder", module: &Module{fileRules: map[string]*config.FileRule{"create": fileRule, "delete": fileRule, "read": fileRule}}},
+		{testName: "Success", path: ps + "folder", module: &Module{fileRules: map[string]*config.FileRule{"create": fileRule1, "delete": fileRule1, "read": fileRule1}}},
+		{testName: "Success", path: ps + "folder" + ps + "file", module: &Module{fileRules: map[string]*config.FileRule{"create": fileRule1, "delete": fileRule1, "read": fileRule1}}},
 
 		//Error Test
 		{testName: "Fail", path: "/NewFolder/file", module: &Module{fileRules: map[string]*config.FileRule{"create": fileRule1, "delete": fileRule1, "read": fileRule1}}},
```
Comment (AliabbasMerchant) [inline modules/auth/file_test.go]: Please replace "/"s in the 35th line also with `ps`

**Comment by AliabbasMerchant** [comment]
Also, @Inconnu08 please fix the `..` error.

> There is a small but dangerous bug. If we specify the filenames as "/../", it allows us to insert into the parent folder, which is dangerous. We must use absolute paths to check.

**Comment by AliabbasMerchant** [comment]
@YourTechBud Looks fine to me. Please have a look

**Comment by YourTechBud** [comment]
Perfect. @Inconnu08 love the fact that you correct the comments as well.

**Pull request merged**